pkg/graveler/ref/manager.go

```diff
@@ -71,6 +71,7 @@ func createBareRepository(tx db.Tx, repositoryID graveler.RepositoryID, reposito
 
 func (m *Manager) CreateRepository(ctx context.Context, repositoryID graveler.RepositoryID, repository graveler.Repository, token graveler.StagingToken) error {
 	firstCommit := graveler.Commit{
+		Version:      graveler.CommitVersion,
```

Comment:
set this inside graveler, not from the outside

Reply:
Agree and disagree with you on this one.
This function has access to all of graveler internals (like `FirstCommitMsg` and even the time is debatable) and the version is not an exception to that. Without setting it here we'll need to add an API (or a const) to graveler named `GetFirstCommit`. Do you like that better?

Reply:
Will add `NewCommit` func that will return the struct initialized so the use of the constant of the current version will be in just one place.